tools/install.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+
+# installation of the ProActive distribution
+ 
+read -e -p "Directory where to install the scheduler: " -i "/opt/proactive" PA_ROOT
+
+mkdir -p $PA_ROOT
+rm -f $PA_ROOT/default
+
+CURRENT_PADIR="$(dirname "$SCRIPT_DIR")"
+PA_FOLDER_NAME="$(basename "$CURRENT_PADIR")"
+
+/usr/bin/cp -f -R $CURRENT_PADIR $PA_ROOT
+
+ln -s -f $PA_ROOT/$PA_FOLDER_NAME $PA_ROOT/default
+
+# creation of the proactive user
+
+read -e -p "Name of the user starting the ProActive service: " -i "proactive" USER
+
+echo "Do you want to create this user ?"
+select yn in "Yes" "No"; do
+    case $yn in
+        Yes ) 
+		useradd -m $USER;
+		passwd $USER; 
+		break;;
+        No ) break;;
+    esac
+done
+
+chown -R $USER:$USER $PA_ROOT/$CURRENT_PA_DIR
+
+# installation of the service script
+
+cp $SCRIPT_DIR/proactive-scheduler /etc/init.d/
+
+read -e -p "Protocol used by the proactive server: " -i "http" PROTOCOL
```

Comment:
You said that it makes sense to specify the options which are allowed between square brackets but you don't did it. Did you forget?